packages/website-frontend/test/helpers/custom-element-fixture.ts

```diff
@@ -0,0 +1,94 @@
+import type { LitElement } from 'lit';
+
+interface CustomElementFixtureOptions {
+  autoConnect: boolean;
+}
+
+const defaultOptions: Readonly<CustomElementFixtureOptions> = Object.freeze({
+  autoConnect: true,
+});
+
+export class CustomElementFixture<TCustomElement extends LitElement> {
```

Comment:
Does this whole thing work with happy-dom? I'd be surprised 😅

Reply:
Haha, nope, it absolutely doesn't 😄 

I was trying to introduce testing in the same way that the report does it, still have to do that